doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -0,0 +1,169 @@
+[[post-installation]]
+== Managing {product-title_short} with OpenShift
+
+This section includes common tasks to manage your {product-title_short} deployment from OpenShift.
```

Comment:
Should we have the full product name here since its the first instance in the section?